GSA/digitalgov.gov
# Hide go.usa.gov from "get started building" sidebar

## Summary

Removes `content/services/service_usagov-short-urls.md` to remove 'go.usa.gov' from sidebar.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-remove-gousa-sidebar/resources/checklist-of-requirements-for-federal-digital-services/?dg)

### Solution

**Before**
<img width="1678" alt="Screen Shot 2023-05-15 at 10 00 15 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/d1a8a0a4-34e9-4f72-8e2a-5a567d878903">

**After**
<img width="1678" alt="Screen Shot 2023-05-15 at 10 00 32 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/3e7ae1b0-402c-43c9-8e5a-7efedd383690">

### How To Test

1. Visit any resources page — [Link](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-remove-gousa-sidebar/resources/checklist-of-requirements-for-federal-digital-services/?dg)
2. Should not see "go.usa.gov" in the "Get started building" sidebar

---

### Dev Checklist
